src/MySql/readme.md

```diff
@@ -0,0 +1,138 @@
+- [NodeJS LTS](https://nodejs.org) (10.15.x LTS preferred)
+  - **Note**: It *will not work* with Node < 10.x. Using 11.x builds may cause issues as they may introduce instability or breaking changes.
+> If you want an easy way to install and update Node, [NVS - Node Version Switcher](../nodejs/installing-via-nvs.md) or [NVM - Node Version Manager](../nodejs/installing-via-nvm.md) is recommended.
+- [AutoRest](https://aka.ms/autorest) v3 beta <br>`npm install -g autorest@beta`<br>&nbsp;
+- PowerShell 6.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g pwsh`<br>&nbsp;
+- .NET Core SDK 2.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g dotnet-sdk-2.2`<br>&nbsp;
+
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+  - $(repo)/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json
+module-version: 0.1.0
+title: MySQL
+subject-prefix: 'MySQL'
+
+directive:
+  - from: mysql.json
+    where: $.definitions.VirtualNetworkRule
+    transform: $['required'] = ['properties']
+  - select: command
+    where:
+      verb: Set
+    set:
+      verb: Update
+  - where:
+      verb: New$|^Set$|^Remove$|^Get|^Update$|^Invoke$
+      subject: Database$|SecurityAlertPolicy$|Administrator$|LocationBasedPerformanceTier$|LogFile$|ExecuteCheckNameAvailability$
+    hide: true
+  - where:
+      verb: New$|Update$
+      subject: Server$
+    hide: true
+  - where:
+     verb: New$
+     variant: ^Create$
+    hide: true
+  - where:
+     verb: New$
+     variant: ^CreateViaIdentity
+    hide: true
+  - where:
+      verb: New$|Update$
+      variant: ^(?!.*?Expanded)
+    hide: true
+  - where:
+      verb: New
+      subject: Configuration
+    hide: true
+  - where:
+      parameter-name: VirtualNetworkSubnetId
+      subject: VirtualNetworkRule
+    set:
+      parameter-name: SubnetId
+  - where:
+      model-name: Server
+    set:
+      format-table:
+        properties:
+          - Name
+          - Location
+          - AdministratorLogin
+          - Version
+          - StorageProfileStorageMb
+          - SkuName
+          - SkuSize
+          - SkuTier
+          - SslEnforcement
+  - where:
+      model-name: Configuration
+    set:
+      format-table:
+        properties:
+          - Name
+          - Value
+  - where:
+      model-name: FirewallRule
+    set:
+      format-table:
+        properties:
+          - Name
+          - StartIPAddress
+          - EndIPAddress
+  - where:
+      parameter-name: StorageProfileBackupRetentionDay
+      subject: Server
+    set:
+      parameter-description: Backup retention days for the server. Day count is between 7 and 35.
+  - from: ServerForCreate.cs
```

Comment:
Shall we need to change the specific file to source-file-csharp? Because it looks not working in some platforms.